docs/design/gh-ost-integration.md

```diff
@@ -160,32 +160,34 @@ Copy: 2/2 100.0%; Applied: 0; Backlog: 0/1000; Time: 2s(total), 1s(copy); stream
 
 In the output, you can see the main components of gh-ost.
 
-**Migrator** is the main schema migration flow manager.
+**Migrator** is the schema migration manager.
 
-**Inspector** reads data from the read-MySQL-server (typically a replica, but can be the master). It is used for gaining initial status and structure, and later also followup on progress and changelog.
+**Inspector** reads data from the read-MySQL-server (typically a replica but can be the master). It gets initial status and structure and later follows up on progress and changelog.
 
 **Streamer** reads data from binary logs and streams it on.
 
-**Applier** connects and writes the applier-server, which is the server where migration happens. Applier is the one to actually write row data and apply binlog events onto ghost table.
+**Applier** connects and writes the applier-server where migration happens. Applier is the one to write row data and apply binlog events onto the ghost table.
 
-**Throttler** collects metrics related to throttling and makes informed decision whether throttling should take place. Here throttling means to pause.
+**Throttler** collects metrics related to throttling and makes the informed decision whether throttling should take place. Here throttling means to pause.
```

Comment:
These are from gh-ost code comments. I might just leave them be.

Reply:
It can still be suboptimal. If it's not a quote per se, I think we should use the optimal expressions.

Reply:
I think both are ok.